docs/faq/contributing.txt

```diff
@@ -97,3 +99,9 @@ entire community, instead of prioritizing the impact on one particular user.
 This doesn't mean that we think your problem is unimportant -- just that in the
 limited time we have available, we will always err on the side of making 10
 people happy rather than making a single person happy.
+
+I'm sure my ticket is absolutely 100% perfect, can I mark it as "Ready For Checkin" myself?
```

Comment:
Not for now! 

I struggle with "checkin" surely it's check-in? When I see it like this I always wonder if it someone's just dropped the "g" of checking?

Reply:
That could just be British people and their 🥰 for hyphens. 🤪